python/lsst/ts/MTAlignment/alignmentModel.py

```diff
@@ -170,6 +172,23 @@ async def laser_off(self):
         data = await self.send_msg("!LST:0")
         return data
 
+    async def set_simulation_mode(self, sim_mode):
+        if sim_mode:
+            data = await self.send_msg("!SET_SIM:1")
+        else:
+            data = await self.send_msg("!SET_SIM:0")
+
+    async def tracker_off(self):
+        """
+        Turns the whole tracker off, must be powered back on "manually"
+
+        Returns
+        -------
+        ACK300 or ERR code
+        """
+        data = await self.send_msg("!LST:2")
+        return data
```

Comment:
Here and in the other commands, consider returning the result of the `await`. This will save time by avoiding to create a variable to be destroyed later and will make the code easier to read:

```
return await self.send_msg("!LST:2")
```